Cargo.lock

```diff
@@ -6108,9 +6259,9 @@ dependencies = [
 
 [[package]]
 name = "tinyfiledialogs"
-version = "3.3.9"
+version = "3.9.1"
```

Comment:
Are you running cargo update? Some of these updates don't look related to egui and winit. I would probably only update the necessary crates for egui and winit as for example this update is known to have issues: #31205

Reply:
Was running into that issue, thanks for the link. Will redo the changes in Cargo.lock, starting from mainline.